app/src/js/types/state.ts

```diff
@@ -303,6 +303,11 @@ export interface Attribute {
   buttonColors: string[]
 }
 
+export interface Category {
```

Comment:
Are you proposing to add more fields to the interface? If only one field exists, can we typedef Category to string directly?

Reply:
Okay, I can just inline this.

Reply:
Moved this type definition to the creation file, since it is only used there.